apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -100,23 +106,31 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     struct Vote {
         uint256 yea;                                        // Voting power for
         uint256 nay;                                        // Voting power against
+        // Note: rename to totalPower?
         uint256 votingPower;                                // Total voting power (based on the snapshot block)
         uint256 settingId;                                  // Identification number of the setting applicable to the vote
         uint256 actionId;                                   // Identification number of the associated disputable action on the attached Agreement
+        // Note: we may want to re-order this state to be based on localized SSTOREs
+        //       start date and snapshot block never change, whereas pausedAt, pauseDuration and status may be changed together
```

Comment:
Not sure about this; just something I was thinking for a bit. It'd be relatively minor, but some of these variables are stored or loaded together at the same time, and we'd want those to be in the same slot.

Reply:
Addressed in 8a6dcf10ed85c12aa184ba3b06eb63d4b8b851e8